charts/govuk-jobs/values.yaml

```diff
@@ -45,3 +45,10 @@ draftContentStoreMongoPostgresCron:
     mongoDbUri: ""
   postgresImport:
     databaseUrlSecretName: "draft-content-store-postgres"
+
+contentStoreMongoPostgresCron:
+  schedule: "15 5 * * *"
```

Comment:
It'd be cool to run these at slightly different times so as to reduce peak resource usage. I'm guessing they don't take hours to run?

Reply:
Sure. 

They need to run after the overnight env_sync job finishes, which IIRC is sometime around 4.15, and the mongo-export part of this completes in a couple of minutes, but the postgres-import part takes about 1h 15m. 

So how about I keep the live one at 5:15, and start the draft one at 6:30 ?

Reply:
Thanks!